client_test.go

```diff
@@ -3956,7 +3956,7 @@ func Test_Client_JobCompletion(t *testing.T) {
 		require.NotNil(updatedJob.FinalizedAt)
 
 		// Make sure the FinalizedAt is approximately ~now:
-		require.WithinDuration(now, *updatedJob.FinalizedAt, time.Second)
+		require.WithinDuration(now, *updatedJob.FinalizedAt, 2*time.Second)
```

Comment:
Totally unrelated, but one that cropped up as an intermittent failure as I reran this PR's test matrix over and over again. Most duration checks use 2s rather than 1s by convention in this file, so this change makes it match up a little better with other assertions.